docs/content/docs/recipes/angular.md

```diff
@@ -1,6 +1,285 @@
 ---
 title: Using Fluid with Angular
 menuPosition: 2
-status: unwritten
-discussion: 5464
+aliases:
+  - "/start/angular-tutorial/"
 ---
+
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other clients almost instantly. You'll also learn how to connect the Fluid data layer with a view layer made in [Angular](https://angular.io).
+
+To jump ahead into the finished demo, check out the [Create React App demo in our FluidExamples repo](https://github.com/microsoft/FluidExamples/tree/main/angular-demo).
+
+The following image shows the time stamp application open in four browsers. Each has a button labeled **click** and beside it a Unix epoch time. The same time is in all four. The cursor is on the button in one browser.
+
+![Four browsers with the Timestamp app open in them.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-1.PNG)
```

Comment:
i think you'll need to create another copy of these images for angular; the existing ones show the tabs titled "React App"

Reply:
@tylerbutler Would you be able to help me setup new screenshots in the same way the react tutorial screenshots were setup?

Reply:
I think @micahgodbolt made the React images. I can upload them to the CDN once you have them.

Reply:
Great, I will email them to you shortly!